src/web/app/components/copy-session-modal/copy-session-modal.component.html

```diff
@@ -9,8 +9,13 @@ <h5 class="modal-title">
   <div class="row">
     <div class="col-12">
       <div class="form-group">
-        <label><b>Name for copied session</b></label>
-        <input id="copy-session-name" type="text" class="form-control" [(ngModel)]="newFeedbackSessionName" [maxlength]="FEEDBACK_SESSION_NAME_MAX_LENGTH">
+        <label><b>Name for copied session*</b></label>
+        <input id="copy-session-name" type="text" class="form-control" [(ngModel)]="newFeedbackSessionName" [maxlength]="FEEDBACK_SESSION_NAME_MAX_LENGTH"
+               required #newSessionName="ngModel">
+        <div [hidden]="newSessionName.valid || (newSessionName.pristine && newSessionName.untouched)" class="invalid-field">
+            <i class="fa fa-exclamation-circle" aria-hidden="true"></i>
+            The field Name for copied session should not be empty.
```

Comment:
Let's change the error message to:

> The field "Name for copied session" should not be empty.

Following #12617, and also because the field name is not entirely capitalised (unlike other fields elsewhere like Session Name)